### PR TITLE
[master][KOGITO-1733] Convert Travel agency test to not use example yaml from…

### PR DESCRIPTION
**Pull request by sutaakar**
… operator repository

Signed-off-by: Karel Suta <ksuta@redhat.com>

https://issues.redhat.com/browse/KOGITO-1733

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by sutaakar** [comment]
@Sgitario No, the YAML files serve as an example for users, to show how can they run Kogito applications.

**Comment by radtriste** [comment]
PR is also addressing directly https://issues.redhat.com/browse/KOGITO-1724 right ?

**Comment by sutaakar** [comment]
Rebased

**Comment by sutaakar** [comment]
Jenkins job passed

File: test/features/deploy_travel_agency.feature [hunk 1]
```diff
@@ -6,15 +6,25 @@ Feature: Deploy Travel agency service and verify its functionality
...
-    And Kogito application "kogito-travel-agency" has 1 pods running within 10 minutes
-    And HTTP GET request on service "kogito-travel-agency" with path "travels" is successful within 1 minutes
-    And Kogito application "kogito-visas" has 1 pods running within 10 minutes
-    And HTTP GET request on service "kogito-visas" with path "visaApplications" is successful within 1 minutes
+    And Deploy quarkus example service "kogito-travel-agency/travels" with native disabled and persistence and events and environment variables
+    | MP_MESSAGING_INCOMING_VISASAPPROVED_BOOTSTRAP_SERVERS                   | |
+    | MP_MESSAGING_INCOMING_VISASREJECTED_BOOTSTRAP_SERVERS                   | |
+    | MP_MESSAGING_OUTGOING_VISAAPPLICATIONS_BOOTSTRAP_SERVERS                | |
+    | MP_MESSAGING_OUTGOING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS  | |
+    | MP_MESSAGING_OUTGOING_KOGITO_USERTASKINSTANCES_EVENTS_BOOTSTRAP_SERVERS | |
```
Comment (ricardozanini) [inline test/features/deploy_travel_agency.feature:14]: Could you try without those env vars?
https://github.com/kiegroup/kogito-examples/pull/184
We will have to wait for @DuncanDoyle PR to get merged though.

File: test/steps/kogitoapp.go [hunk 2]
```diff
@@ -94,6 +95,21 @@ func (data *Data) deployQuarkusExampleServiceWithNativeAndPersistenceAndEvents(c
 	return framework.DeployService(data.Namespace, framework.GetDefaultInstallerType(), kogitoApp)
 }
 
+func (data *Data) deployQuarkusExampleServiceWithNativeAndPersistenceAndEventsAndEnvironmentVariables(contextDir, native string, dt *gherkin.DataTable) error {
```
Comment (ricardozanini) [inline test/steps/kogitoapp.go:98]: In the future, consider using a pattern like:

```go
deploy().
  Service(quarkusExample).
  Native().
  Persistence().
  Events().
  EnvVars(map)
```

File: test/features/deploy_travel_agency.feature (hunk 1 above)
Comment (sutaakar) [reply test/features/deploy_travel_agency.feature]: It works, PR updated.

**Comment by sutaakar** [comment]
The Cucumber tests passed

**Pull request merged**